src/Ledger/PParams.lagda

```diff
@@ -44,68 +44,64 @@ record PoolThresholds : Set where
 record PParams : Set where
   field
 \end{code}
-\emph{Network group}
+\emph{Network group}\vskip-3mm
```

Comment:
The formatting mod `\vskip-3mm` (here and a few places below) merely puts the parameter group name closer to its parameters and helps to ensure that the entire figure fits on a single page.